pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "cert-operator"
 	source      string = "https://github.com/giantswarm/cert-operator"
-	version            = "n/a"
+	version            = "1.1.0-dev"
```

Comment:
According to the process it should be previous version + 0.0.1-dev:

```suggestion
	version            = "1.0.1-dev"
```